src/components/DiagramFrame/utils.ts

```diff
@@ -1,14 +1,24 @@
-import { IGitBranch, ILookmlModelExplore } from "@looker/sdk/lib/sdk/3.1/models"
+import { IGitBranch, ILookmlModel, ILookmlModelExplore } from "@looker/sdk/lib/sdk/3.1/models"
 import { SelectOptionProps, theme } from "@looker/components"
-import { OVERRIDE_KEY_SUBTLE } from "../../utils/constants"
+import { OVERRIDE_KEY_SUBTLE, DIAGRAM_IGNORED_MODELS } from "../../utils/constants"
+import { ExploreDropdown } from "./types"
 
+/**
+ * styles the view list according to visibility status
```

Comment:
Not really. Style implies something getting styled. This is retuning some kind of color (or not) that I expect is used for styling. The function name is confusing as well (don't need to change the name as part of this PR though).